doc/sphinx/changes.rst

```diff
@@ -8,6 +8,977 @@ Recent changes
+  `ConstructivePower`.
+
+  .. warning:: The constructive reals modules are marked as experimental.
+
+  (`#11725 <https://github.com/coq/coq/pull/11725>`_,
+  `#12287 <https://github.com/coq/coq/pull/12287>`_
+  and `#12288 <https://github.com/coq/coq/pull/12288>`_,
+  by Vincent Semeria).
+- **Removed:**
+  Type `RList` has been removed.  All uses have been replaced by `list R`.
+  Functions from `RList` named `In`, `Rlength`, `cons_Rlist`, `app_Rlist`
+  have also been removed as they are essentially the same as `In`, `length`,
+  `app`, and `map` from `List`, modulo the following changes:
+
+    - `RList.In x (RList.cons a l)` used to be convertible to
+      `(x = a) \\/ RList.In x l`,
+      but `List.In x (a :: l)` is convertible to
+      `(a = x) \\/ List.In l`.
+      The equality is reversed.
+    - `app_Rlist` and `List.map` take arguments in different order.
+
+  (`#11404 <https://github.com/coq/coq/pull/11404>`_,
+  by Yves Bertot).
+- **Added:**
+  inverse trigonometric functions `asin` and `acos` with lemmas for
+  the derivatives, bounds and special values of these functions; an
+  extensive set of identities between trigonometric functions and
+  their inverse functions; lemmas for the injectivity of sine and
+  cosine; lemmas on the derivative of the inverse of decreasing
+  functions and on the derivative of horizontally mirrored functions;
+  various generic auxiliary lemmas and definitions for `Rsqr`, `sqrt`,
+  `posreal` and others
+  (`#9803 <https://github.com/coq/coq/pull/9803>`_,
+  by Laurent Théry and Michael Soegtrop).
+
+Extraction
+^^^^^^^^^^
+
+- **Added:**
+  Support for better extraction of strings in OCaml and Haskell:
+  `ExtOcamlNativeString` provides bindings from the Coq `String` type to
+  the OCaml `string` type, and string literals can be extracted to literals,
+  both in OCaml and Haskell (`#10486
+  <https://github.com/coq/coq/pull/10486>`_, by Xavier Leroy, with help from
+  Maxime Dénès, review by Hugo Herbelin).
+- **Fixed:**
+  In Haskell extraction with ``ExtrHaskellString``, equality comparisons on
+  strings and characters are now guaranteed to be uniquely well-typed, even in
+  very polymorphic contexts under ``unsafeCoerce``; this is achieved by adding
+  type annotations to the extracted code, and by making ``ExtrHaskellString``
+  export ``ExtrHaskellBasic`` (`#12263
+  <https://github.com/coq/coq/pull/12263>`_, by Jason Gross, fixes `#12257
+  <https://github.com/coq/coq/issues/12257>`_ and `#12258
+  <https://github.com/coq/coq/issues/12258>`_).
+
+Reference manual
+^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  The reference manual has been restructured to get a more logical
+  organization.  In the new version, there are fewer top-level
+  chapters, and, in the HTML format, chapters are split into smaller
+  pages.  This is still a work in progress and further restructuring
+  is expected in the next versions of Coq
+  (`CEP#43 <https://github.com/coq/ceps/pull/43>`_, implemented in
```

Comment:
Just a thought: it might friendlier to link to `https://github.com/coq/ceps/pull/43/files` which shows the text of the CEP.  Or, better, the formatted document if that's accessible.  The latter would be good for the 8.12 announcement email, which can include a CEP link for further information.

Reply:
The top post of the CEP contains the link to the rendered version already.